portability-worker/src/main/java/org/dataportabilityproject/worker/JobMetadata.java

```diff
@@ -56,6 +56,14 @@ static void init(
     importService = initImportService;
   }
 
+  static void reset() {
```

Comment:
If we keep this around, let's synchronize on this

Reply:
Good point, I think we should get rid of this, let me put a TODO in there.